L1Trigger/L1TGlobal/python/simGtStage2Digis_cfi.py

```diff
@@ -15,6 +15,7 @@
     TauInputTag = cms.InputTag("simCaloStage2Digis"),
     JetInputTag = cms.InputTag("simCaloStage2Digis"),
     EtSumInputTag = cms.InputTag("simCaloStage2Digis"),
+    EtSumZdcInputTag = cms.InputTag("etSumZdcProducer"),
```

Comment:
I did not make this change, in part because this is at the level of the python instance of the producer and I was not confident I would be able to fix globally at this time; I will flag however for future PRs

Unchanged in latest commit